### ISSUE TITLE
Łódzkie -  miasta

### ISSUE THREAD

**Issue by bdkacz** [szczepienia/szczepienia.github.io#28]
Tak wiem, nie miało być próśb o miasta, ale może dało by się dodać chociaż wszystkie miasta powiatowe ? Konkretnie o Opoczno mi chodzi :)

**Comment by szczepienia** [comment]
Tak, jeśli nie ma większych miast to nie ma terminów w najbliszym czasie.

Ogólnie to jeśli miasto jest na liście w danej stronie (tam jest taka lista "Wyszukujemy w miastach") to jeśli takiego miasta nie ma to nie ma terminów do końca maja (obecnie, bo dodają terminy codziennie).
